pyspark/bigdl/util/tf_utils.py

```diff
@@ -63,6 +66,74 @@ def convert(input_ops, output_ops, byte_order, bigdl_type):
 
     return model
 
+def export_checkpoint(checkpoint_path):
+    """
+    Export variable tensors from the checkpoint files.
+
+    :param checkpoint_path: tensorflow checkpoint path
+    :return: dictionary of tensor. The key is the variable name and the value is the numpy
+    """
+    reader = tf.train.NewCheckpointReader(checkpoint_path)
+
+    # Get tensor name list
+    tensor_names = filter(lambda n: n!='global_step',
+                          reader.get_variable_to_shape_map().keys())
+    # Prepare key-value dictionary
+    tensors = {}
+    for tn in tensor_names:
+        tensors[tn] = reader.get_tensor(tn)
+
+    return tensors
+
+def save_variable_bigdl(tensors, target_path, bigdl_type = "float"):
+    """
+    Save a variable dictionary to a Java object file, so it can be read by BigDL
+
+    :param tensors: tensor dictionary
+    :param target_path: where is the Java object file store
+    :return: nothing
+    """
+    jtensors = {}
+    for tn in tensors.keys():
+        jtensors[tn] = JTensor.from_ndarray(tensors[tn])
+        
+    callBigDlFunc(bigdl_type, "saveTensorDictionary", jtensors, target_path)
+
+
+def dump_model(path, bigdl_type="float"):
```

Comment:
Should we consider let user pass in the `Graph` and `Session` object explicitly, so that they can pass in other `Graph` besides the default graph and also pass in a session that restored from a existing pre-trained checkpoint if he/she want to.